ts41/get.d.ts

```diff
@@ -0,0 +1,82 @@
+import {Split} from '../source/split';
+import {Integers} from '../source/utilities';
+
+/**
+ * Gets a deeply-nested property from an object, like lodash's `get` method.
+ *
```

Comment:
Don't prefix the doc comments with `*`. See the other type's doc comments.

Reply:
👍 have removed the prefixes for consistency, but I wonder if they should be added to the other comments (in a follow-up)? VSCode seems to prefer the stars:

Without stars:

![image](https://user-images.githubusercontent.com/15040698/100376735-08187d80-2fde-11eb-8997-35324f7ed81e.png)

With stars:

![image](https://user-images.githubusercontent.com/15040698/100376826-2da58700-2fde-11eb-8ca2-287624a4ba59.png)

(the formatting gets flattened without the stars)

Either way, totally agree this type should be inline with all the other existing ones right now.